system-parachains/coretime/coretime-kusama/src/coretime.rs

```diff
@@ -253,8 +214,5 @@ impl pallet_broker::Config for Runtime {
 	type WeightInfo = weights::pallet_broker::WeightInfo<Runtime>;
 	type PalletId = BrokerPalletId;
 	type AdminOrigin = EnsureRoot<AccountId>;
-	#[cfg(feature = "runtime-benchmarks")]
-	type PriceAdapter = pallet_broker::Linear;
-	#[cfg(not(feature = "runtime-benchmarks"))]
-	type PriceAdapter = PriceAdapter;
```

Comment:
for my education: are benchmarks updated in a separate PR? what is the current process? #324?